cmd/flowrunner/main.go

```diff
@@ -26,9 +25,8 @@ type Output struct {
 }
 
 type FlowTest struct {
-	Extra        json.RawMessage        `json:"extra,omitempty"`
-	CallerEvents []*utils.TypedEnvelope `json:"caller_events"`
-	Outputs      []json.RawMessage      `json:"outputs"`
+	CallerEvents [][]*utils.TypedEnvelope `json:"caller_events"`
```

Comment:
Until now we've used caller_events as a list of events where each event corresponds to a resume - but now we need initial events (e.g. set_extra), and we want to support multiple events per start/resume